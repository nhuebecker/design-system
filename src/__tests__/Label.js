import React from 'react'
import renderer from 'react-test-renderer'
import { Label } from '..'

describe('Label', () => {
  test('it renders', () => {
    const json = renderer.create(<Label />).toJSON()
    expect(json).toMatchSnapshot()
  })
})
