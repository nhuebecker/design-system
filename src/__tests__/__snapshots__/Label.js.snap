// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label it renders 1`] = `
.c1 {
  font-size: 18px;
  margin: 0px;
  color: #384046;
}

.c0 {
  display: block;
}

.c0 a {
  color: #2d9ee4;
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

<label
  className="c0 c1"
  color="black"
  fontSize={2}
/>
`;
